conf/uppmax.config

```diff
@@ -54,26 +60,18 @@ process {
   scratch = '$SNIC_TMP'
 }
 
-params {
-  save_reference = true
-
-  max_memory = 500.GB
-  max_cpus = 16
-  max_time = 240.h
-  // illumina iGenomes reference file paths on UPPMAX
-  igenomes_base = '/sw/data/igenomes/'
-}
-
-if (hostname ==~ "s[0-9][0-9]*") {
+if (hostname.startsWith("s")) {
```

Comment:
This one breaks bianca IIRC (where IIRC nodes will be project name-nodenumber, e.g. sens2021500-001 or similar).